redfish-core/lib/log_services.hpp

```diff
@@ -6970,6 +7040,7 @@ inline void fillSystemHardwareIsolationLogEntry(
         "/redfish/v1/Systems/system/LogServices/HardwareIsolation/Entries/{}",
         dbusObjIt->first.filename());
     entryJson["Id"] = dbusObjIt->first.filename();
+    entryJson["MessageId"] = "OpenBMC.0.6.GardRecord";
```

Comment:
It looks like `GardRecord` entry needs to be added to `redfish-core/include/registries/openbmc.json`